php/thankYou.php

```diff
@@ -122,45 +305,64 @@
                 echo "<p>T-shirt Size: $tShirt</p>";
 
                 //address
-                echo "<p>Address: ".$street." ".$city.", ".$state." ".$zip."</p>";
+                echo "<h5>Address</h5>";
+                echo "<p>Street: $street</p>";
+                echo "<p>City: $city</p>";
+                echo "<p>State: $state</p>";
+                echo "<p>Zip Code: $zip</p>";
 
                 //interests
                 echo "<h5>Interests</h5>";
-                if(!isset($_POST["interests"])){
-                    echo "<p>None selected</p>";
-                }
-                else {
-                    echo "<ul>";
-                    foreach($_POST["interests"] as $interests) {
-                        if($interests === "other"){
-                            echo "<li>Other Skills: ".$_POST["interestsText"]."</li>";
-                            $mailInterests .= ", {$_POST["interestsText"]}";
-                        }
-                        else {
-                            echo "<li>" . $interests . "</li>";
-                            $mailInterests .=  ", $interests";
+                if(validSelectWText($interstArray,$_POST["interests"])) {
```

Comment:
"Notice:  Undefined index: interests..." is on the page when I submit an empty form while debugging.